internal/commands/user/disable.go

```diff
@@ -47,12 +47,16 @@ func (cmd *CommandDisable) Handler(profile *cli.Profile, ui terminal.UI) error {
 		return appErr
 	}
 
-	users, usersErr := cmd.realmClient.FindUsers(app.GroupID, app.ID, realm.UserFilter{IDs: cmd.inputs.Users})
-	if usersErr != nil {
-		return usersErr
+	resolved, resolvedErr := cmd.inputs.resolveUsers(cmd.realmClient, app.GroupID, app.ID)
```

Comment:
now that this original function will be renamed to `findUsers`, each one of these `disableInputs`/`revokeInputs`/etc structs should then implement their own:

```go
func (i disableInputs) resolveUsers(ui terminal.UI, realmClient realm.Client, app realm.App) ([]realm.User, error) {
  found, foundErr := i.findUsers(realmClient, app)
  // ...
  
  return i.selectUsers(ui, users, "disable")
}
```

with this approach, you can go back to calling these `users, usersErr` in the command code where you call `cmd.inputs.resolveUsers(ui, cmd.realmClient, app)`